test/unit/constants_test.rb

```diff
@@ -28,10 +28,21 @@ class ConstantsTest < ActiveSupport::TestCase
     assert string.scan(Callouts.const_get(:HASHTAG)).length == 3
   end
 
-  test 'hashtagnumber regex should match non-initial hash' do 
+  test 'hashtagnumber regex should match non-initial hash' do
     string = "hello #123 #hello " ;
     assert string.scan(Callouts.const_get(:HASHTAGNUMBER)).length == 1
     expect = "hello REPLACE #hello " ;
     assert_equal expect, string.gsub(Callouts.const_get(:HASHTAGNUMBER), ' REPLACE') ;
   end
+
+  test 'checkbox regex should match a checked checkbox' do
+    string = "* [x]"
+    assert string.scan(Callouts.const_get(:CHECKBOX))[0][0] == "x"
+  end
+
+  test 'checkbox regex should match an unchecked checkbox' do
+    string = "* [ ]"
+    assert string.scan(Callouts.const_get(:CHECKBOX))[0][0] == " "
```

Comment:
These look great. Maybe want to test doing a substitution and seeing that it can be converted in the revision unit tests? 

Thank you!!!